File: Week6/sentimental-cash/cash.py
from cs50 import get_float


def num_of_change(total):
    total = int(total * 100)
    change = 0

    change += total // 25
    total %= 25

    change += total // 10
    total %= 10

    change += total // 5
    total %= 5

    change += total

    return change


def main():
    total = get_float("Change: ")
    if total < 0:
        total = get_float("Change: ")

    change = num_of_change(total)
    print(change)


main()

File: Week1/cash/cash.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int left;
    do
    {
        left = get_int("Change owed: ");
    }
    while (left < 0);

    int total = 0;
    total += (left / 25);
    left %= 25;

    total += (left / 10);
    left %= 10;

    total += (left / 5);
    left %= 5;

    total += (left / 1);

    printf("%i\n", total);
}
